opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/metrics/MultiMetricProducer.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim.metrics;
+
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.export.MetricProducer;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/** Class that wraps multiple metric producers into one. */
```

Comment:
We have a similar class for exporter, should this be in the SDK as an official class? Or only has value for metrics shim since otherwise only the OTel SDK produces metrics?

Reply:
Which class are you talking about?   I think I use the composite pattern in a few places, but not this one just yet.

Regarding longer term plans, post SDK stable launch, I hope to expose (via specification) a hook on SdkMeterProvider to register something like this for adapting external metric systems.  For now, I just wanted better code coverage and there was a quick win for a nicer hook with the new metrics SDK.  (Note: Whatever we do here + in Go will drive the [OpenCensus compatiblity specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/opencensus.md#opencensus-compatibility)